frontend/src/lib/components/SaveToDashboard/SaveToDashboardModal.js

```diff
@@ -29,11 +29,14 @@ const saveToDashboardModalLogic = kea({
         },
     }),
 })
+
 const radioStyle = {
     display: 'block',
-    height: '30px',
-    lineHeight: '30px',
+    lineHeight: 30,
+    overflow: 'hidden',
+    whiteSpace: 'normal',
 }
+
```

Comment:
This should work nicer, breaking the label into multiple lines if it ends up being too long, instead of hiding the overflow.